### PR TITLE
Bump jdk-11

### PR DESCRIPTION
**Pull request by g4s8**
#1 - updated jdk from 1.8 to 11:
 - Oracle stops the support for jdk-8, it won't be updated and may contain critical and security bugs
 - openjdk-11 has a lot of performance improvements and new garbage collectors - it's faster 
 - it will be easier to update to new openjdk versions (e.g. 13, 14)

### CODE REVIEW

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 updated pdd config, can we try again?

**Comment by yegor256** [comment]
@rultor merge

File: .pdd [hunk 1]
```diff
@@ -4,6 +4,6 @@
 --exclude target/**/*
 --exclude src/main/resources/images/**/*
 --exclude .idea/**/*
---rule min-words:20
+--rule min-words:15
```
Comment (Sammers21) [inline .pdd]: How this change is related to this Pull Request?

Comment (g4s8) [reply .pdd]: @Sammers21 it's needed to allow submitting smaller puzzles than 20 words, see todos in `pom.xml`

Comment (Sammers21) [reply .pdd]: But the Pull Request name is `Bump jdk-11`, which is not related to puzzlers in any way

**Comment by Sammers21** [comment]
@g4s8 , close this pull request please

**Comment by 0crat** [comment]
This job is not in scope

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#2` is not assigned, can't get performer

**Pull request closed without merging**